src/main/java/Mona.java

```diff
@@ -0,0 +1,54 @@
+import mona.input.Parser;
+import mona.input.Ui;
+import mona.output.ConsolePrint;
+import mona.manager.TaskList;
+import mona.util.Constants;
+
+import mona.storage.Storage;
+
+/**
+ * The main class of the Mona application, responsible for initializing the application.
+ */
+public class Mona {
+
+    private Ui ui;
+    private Storage storage;
+    private TaskList taskList;
+
+    /**
+     * Constructor for Mona. Initializes the user interface, storage,
+     * and task list components of the application.
+     */
+    public Mona() {
+        this.ui = new Ui();
+        this.storage = new Storage(Constants.DATA_FILE_PATH);
+        this.taskList = new TaskList((this.storage).loadData());
+    }
+    /**
```

Comment:
need some vertical spacing between the 2 methods to improve readability.
pls fix all instances.